### PR TITLE
Adding nmea_parser package to documentation index for distro<humble>

### PR DESCRIPTION
**Pull request by HundredGB**
Please add the following dependency to the rosdep database.

## Package name: nmea_parser

A node that receives nmea data (GGA, RMC, HDT), parses it, and then publishes it

## Package Upstream Source: https://github.com/HundredGB/nmea_parser

TODO link to source repository

## Purpose of using this: for education

TODO Replace. This dependency is being used for this reason. This is why I think it's valuable to be added to the rosdep database.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - not available
- Ubuntu: https://packages.ubuntu.com/
  - REQUIRED
- Fedora: https://packages.fedoraproject.org/
  - not available
- Arch: https://www.archlinux.org/packages/
  - not available
- Gentoo: https://packages.gentoo.org/
  - not available
- macOS: https://formulae.brew.sh/
  - not available
- Alpine: https://pkgs.alpinelinux.org/packages
  -not available
- NixOS/nixpkgs: https://search.nixos.org/packages
  - not available
- openSUSE: https://software.opensuse.org/package/
  -not available
- rhel: https://rhel.pkgs.org/
  - not available

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME : humble

# The source is here:

https://github.com/HundredGB/nmea_parser

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Comment by sloretz** [comment]
[Holding for Humble sync](https://discourse.ros.org/t/preparing-for-humble-sync-and-patch-release-2024-08-06/38909)

**Comment by HundredGB** [comment]
The issue has been fixed! Please re-register.

**Review by audrow — APPROVED** [review]

**Review by audrow — COMMENTED** [review]
It looks like CI is failing due to a formatting issue.

File: humble/distribution.yaml [hunk 1]
```diff
@@ -5170,6 +5170,19 @@ repositories:
       url: https://github.com/ros-drivers/nmea_navsat_driver.git
       version: ros2
     status: maintained
+  nmea_parser:
+    doc:
+      type: git
+      url: https://github.com/HundredGB/nmea_parser.git
+      version: main
+    release:
+    tags:
```
Comment (clalancette) [inline humble/distribution.yaml:5179]: This should be removed:

```suggestion
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by audrow** [comment]
Closing because it's stale. Feel free to reopen.

**Pull request closed without merging**